### PR TITLE
Cannot use conda-build 1.20.1 PY34Win64

### PR DESCRIPTION
**Pull request by ocefpaf**
Use conda-build `1.20.1` everywhere but `PY34` and `Win64`.

### CODE REVIEW

File: recipe/run_conda_forge_build_setup_win [hunk 1]
```diff
@@ -8,5 +8,5 @@ conda install -n root --yes --quiet jinja2 anaconda-client
 
 # see https://github.com/conda-forge/staged-recipes/pull/750#issuecomment-225165530
 conda install -n root --yes --quiet conda-build=1.20.1
-
+if "%TARGET_ARCH%" == "x64" if "%CONDA_PY%" == "34" conda install -n root --yes --quiet conda-build=1.20.0
```
Comment (jakirkham) [inline recipe/run_conda_forge_build_setup_win:16]: This is fine. Could we please get the comment back though?

Comment (jakirkham) [inline recipe/run_conda_forge_build_setup_win:16]: Sorry, could have been clearer. Was meaning this [comment](https://github.com/conda-forge/conda-forge-build-setup-feedstock/blob/646cd541768932c480724b6d41801bd973b434f6/recipe/run_conda_forge_build_setup_win#L9-L11).

Comment (ocefpaf) [inline recipe/run_conda_forge_build_setup_win:16]: What comment? Above there is one mentioning that we need to fix `conda-build-all`, and all the issues with 1.20.1 are already fixed in the latest `conda-build`, no? (Actually we never tested it as the pinning here remained...)

Comment (jakirkham) [inline recipe/run_conda_forge_build_setup_win:16]: Yeah, I realized that was unclear and added a link in a comment that appears to have shown up just before yours.

**Pull request merged**